src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download %s</string>
+  <string name="error_syncing_format">Could not sync %s</string>
+  <string name="error_validating_format">Could not validate %s</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">Downloaded %s</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all future versions</string>
+  <string name="ignore_this_update">Ignore this</string>
```

Comment:
I still think that original strings should be used. Especially "ignore this" is weird, ignore what "this"? Since I'm using "update" everywhere, it's good to keep it as it is.

Reply:
What about just "Ignore"?

Reply:
<p>
<img src="https://user-images.githubusercontent.com/24494863/86581575-cc6a1200-bf6f-11ea-8ef3-7168409d2b60.png" width="45%" />
<img src="https://user-images.githubusercontent.com/24494863/86581622-dd1a8800-bf6f-11ea-9349-9103e13033eb.png" width="45%" />
</p>

There are two buttons for different purposes (to ignore, well, "all updates", and ignore, hmm, "this update"). Just "ignore" is not suitable at all. First screenshot is from Foxy Droid, second screenshot is from the official client. This is why I want to revert to "ignore all updates" and "ignore this update". They're perfectly suitable and used by both clients. There is nothing wrong with them to change.

Reply:
OK, will revert. The distinction between ignoring updates for all packages, and "ignoring all future updates of this package" is not clear though.